cli/wallet/wallet_test.go

```diff
@@ -822,6 +863,31 @@ func TestOfflineSigning(t *testing.T) {
 			"--in", txPath)
 	})
 	e.CheckTxPersisted(t)
+
+	t.Run("await 1/1 multisig", func(t *testing.T) {
+		args := []string{"neo-go", "wallet", "nep17", "transfer",
+			"--rpc-endpoint", "http://" + e.RPC.Addresses()[0],
+			"--wallet", walletPath,
+			"--from", testcli.ValidatorAddr,
+			"--to", w.Accounts[0].Address,
+			"--token", "NEO",
+			"--amount", "1",
+			"--force",
+		}
+		e.Run(t, append(args, "--out", txPath)...)
+
+		e.In.WriteString("one\r")
+		e.Run(t, "neo-go", "wallet", "sign",
+			"--wallet", testcli.ValidatorWallet, "--address", testcli.ValidatorAddr,
+			"--in", txPath, "--out", txPath)
+
+		e.Run(t, "neo-go", "wallet", "sign",
+			"--rpc-endpoint", "http://"+e.RPC.Addresses()[0],
+			"--wallet", walletPath, "--address", testcli.ValidatorAddr,
+			"--in", txPath, "--await")
+		e.CheckTxPersisted(t)
```

Comment:
CheckAwaitableTxPersisted.

We write unit tests to check that command works properly. If it doesn't work properly, then there's an error either in test or a bug in command.